Misc/NEWS.d/next/Library/2023-10-29-15-35-57.gh-issue-102221.fQnOaT.rst

```diff
@@ -0,0 +1 @@
+Optimize :func:`math.lcm` by swapping arguments (smaller by absolute value --- first).
```

Comment:
This is an implementation detail, not useful to the user.

Reply:
Polished, thanks!